chromium_src/chrome/browser/ui/webui/settings/metrics_reporting_handler.cc

```diff
@@ -3,12 +3,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#if defined(BRAVE_CHROMIUM_BUILD)
 #define GOOGLE_CHROME_BUILD
```

Comment:
Unrelated to this PR, but we should probably include all headers from the original file here.

Reply:
I'll prepare a separate PR for these changes. Thanks

Reply:
On a second thought, I think the change is small enough we can include it here (I thought there would be many other cases defining `GOOGLE_CHROME_BUILD` in other chromium_src overrides, but turns out it's only these two files). Let me know if you prefer it on a separate PR, though, but otherwise I'm merging that into this PR.